Neos.Flow/Classes/Persistence/Doctrine/Mapping/Driver/FlowAnnotationDriver.php

```diff
@@ -178,9 +178,13 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
          * @var ORM\ClassMetadata $metadata
          */
 
-        $class = $metadata->getReflectionClass();
-        $classSchema = $this->getClassSchema($class->getName());
-        $classAnnotations = $this->reader->getClassAnnotations($class);
+        try {
+            $class = $metadata->getReflectionClass();
+            $classSchema = $this->getClassSchema($class->getName());
+            $classAnnotations = $this->reader->getClassAnnotations($class);
+        } catch (ClassSchemaNotFoundException $exception) {
```

Comment:
Just wondering: Can all of the three lines lead to the `ClassSchemaNotFoundException`? Otherwise I would suggest to wrap the `try/catch` only around the "expected" exceptional cases

Reply:
Should be only the middle one.

Reply:
It is the middle one. I still like the grouping, those lines are really closely related. "Splitting" them by a try/catch seemed confusing.